docs/readmes/orc8r/upgrade_1_4.md

```diff
@@ -0,0 +1,66 @@
+---
+id: upgrade_1_4
+title: Upgrade to v1.4
+hide_title: true
+---
+
+# Upgrade to v1.4
+
+This guide covers upgrading Orchestrator deployments from v1.3 to v1.4.
+Upgrades which skip versions (e.g. v1.2 to v1.4) are not explicitly documented
+or supported at this time.
+
+The v1.4 upgrade follows a standard procedure:
+
+- Upgrade the Terraform modules
+- Bump version numbers
+
+## Prerequisites
+
+Build and publish the application containers and helm charts on the head of the
+release branch by following the documentation [here](https://docs.magmacore.org/docs/orc8r/deploy_build).
+
+If you are using local Terraform state (the default), ensure all Terraform state files (i.e. [`terraform.tfstate`](https://www.terraform.io/docs/state/index.html)) are located in your working directory before proceeding. This means `terraform show` should list existing state (rather than outputting `No state`).
```

Comment:
- Nit: "i.e." is unnecessary
- "rather than outputting ..." can use comma rather than parenthesis
- Line too long